frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -215,9 +211,7 @@ export const funnelLogic = kea<funnelLogicType>({
 
                         return { results: result.result, timeConversionResults }
                     } catch (e) {
-                        if (!isBreakpoint(e)) {
-                            insightLogic.actions.endQuery(queryId, ViewType.FUNNELS, null, e)
-                        }
+                        insightLogic.actions.endQuery(queryId, ViewType.FUNNELS, null, e)
                         return EMPTY_FUNNEL_RESULTS
```

Comment:
Ran into a bug where queries weren't ending for some graph loads. Shouldn't we always end queries on catching errors?

Reply:
This will cancel the preceding request if one is active, right? I think that's what we want — always have the latest request's values. Totally different library than Kea, but in Redux-Saga this behavior would be called [takeLatest](https://redux-saga.js.org/docs/advanced/Concurrency/).

Reply:
That makes a lot of sense. I made this change b/c for some reason the `showTimeoutMessage` var seems to be persisting across sync calls, but I think I should put this back where I found it then.

Reply:
The breakpoints were actually inspired by `takeLatest` and try to imitate the same feature :). redux-saga offers many more controls though.